.gitignore

```diff
@@ -19,3 +19,5 @@ target
 .idea
 
 test/orbreinvoke/**/*~
+
+scratch
```

Comment:
test run outputs, diffs, whatever. As I said in the squashed commit, transient gradeaux. I wanted to be sure I didn't accidentally git add that stuff.